pallets/charity/Cargo.toml

```
@@ -1,9 +1,20 @@
 [package]
 name = "charity"
-version = "2.0.0-alpha.8"
+version = "2.0.0-rc2"
 authors = ["4meta5"]
+repository = 'https://github.com/substrate-developer-hub/recipes'
 edition = "2018"
 license = "GPL-3.0-or-later"
+description = "A charitable organization that accepts donations and imbalances"
```

"imbalances"? Not sure I understand what this means.

---

Currency Imbalances are a cool part of Substrate's Balances pallet. Some docs are here https://crates.parity.io/frame_support/traits/trait.Imbalance.html

The technique is described in the writeup https://substrate.dev/recipes/3-entrees/charity.html#imbalances

I agree it doesn't make sense here without already knowing what imbalances are
```suggestion
description = "A charitable organization that collects funds from donations and imbalances"
```